packages/amplication-git-utils/src/git-provider.interface.ts

```diff
@@ -30,7 +30,7 @@ export interface GitProvider {
   init(): Promise<void>;
   getGitInstallationUrl(amplicationWorkspaceId: string): Promise<string>;
   getCurrentOAuthUser(accessToken: string): Promise<CurrentUser>;
-  getAccessToken(authorizationCode: string): Promise<OAuthData>;
+  getOAuth2FlowData(authorizationCode: string): Promise<OAuthData>;
```

Comment:
what do you think of the following?
```suggestion
  getOAuthTokens(authorizationCode: string): Promise<OAuthTokens>;
```